### PR TITLE
Add integration test for provisioning/deleting many PVs.

### PR DESCRIPTION
**Pull request by jsafrane**
The test is configurable by KUBE_INTEGRATION_PV_OBJECTS for load tests, 100 objects are created by default.

@kubernetes/sig-storage

### CODE REVIEW

**Comment by jsafrane** [comment]
This test is flaky because of #27004

**Comment by jsafrane** [comment]
Marking as "do not merge" until #27004 is fixed

File: test/integration/persistent_volumes_test.go [hunk 1]
```diff
@@ -384,6 +384,85 @@ func TestPersistentVolumeMultiPVsPVCs(t *testing.T) {
...
 }
 
+// TestPersistentVolumeProvisionMultiPVCs tests provisioning of many PVCs.
+// This test is configurable by KUBE_INTEGRATION_PV_* variables.
+func TestPersistentVolumeProvisionMultiPVCs(t *testing.T) {
+	_, s := framework.RunAMaster(t)
+	defer s.Close()
+
+	deleteAllEtcdKeys()
+	testClient, binder, watchPV, watchPVC := createClients(t, s)
```
Comment (rootfs) [inline test/integration/persistent_volumes_test.go]: nit, binder -> controller

**Comment by rootfs** [comment]
LGTM

**Comment by dchen1107** [comment]
Applied lgtm label per reviewers' comments above.

**Comment by dchen1107** [comment]
Decision was made too soon. Removed lgtm label since #27004 was not fixed. I removed 1.3 milestone for this pr.

**Comment by jsafrane** [comment]
#27004 is fixed, marking as lgtm.

**Comment by jsafrane** [comment]
Rebased

**Pull request merged**